src/data/faq.json

```
@@ -2035,6 +2035,42 @@
     },
     "glossary": {
         "glossaryHeadline": "Glossary",
+        "optionalText": "",
+        "glossaryWords": {
+            "B": [
+                {
+                    "term": "Business rules",
+                    "description": "For the introduction of digital vaccination certificates, the eHealth network of the European Comission and the EU member states reconciled that every country compiles the individual rules which have to be checked for transnational travelling. These \"business rules\" have two categories:
+                    <ul><li>Invalidation rules of the country of administration, which decide under which circumstances a certificate is rejected</li><li>Acceptance rules of the country of arrival, which must be fulfilled to enter the country</li></ul>",

```

```suggestion
                    "description": "For the introduction of digital vaccination certificates, the eHealth network of the European Comission and the EU member states reconciled that every country compiles the individual rules which have to be checked for transnational travelling. These \"business rules\" have two categories:<ul><li>Invalidation rules of the country of administration, which decide under which circumstances a certificate is rejected</li><li>Acceptance rules of the country of arrival, which must be fulfilled to enter the country</li></ul>",
```
Remove new line which causes a syntax error.